charts/Makefile

```diff
@@ -0,0 +1,37 @@
+HELM ?= helm
+BUILD_DIR ?= bin
+OUTPUT_DIR ?= $(BUILD_DIR)
+CHART_DIR := $(BUILD_DIR)/gadget
+CHART_VERSION := 0.1.0
+IMAGE_TAG ?= $(shell ../tools/image-tag branch)
+
+clean:
+	rm -rf $(BUILD_DIR)
+
+build: clean
+	@echo "Building chart:"
+	mkdir -p $(BUILD_DIR)
+	cp -r gadget $(BUILD_DIR)
+	cp -r ../pkg/resources/crd/bases $(CHART_DIR)/crds
+	mv $(CHART_DIR)/Chart.yaml.tmpl $(CHART_DIR)/Chart.yaml
+	sed -i "s/%VERSION%/$(CHART_VERSION)/g" $(CHART_DIR)/Chart.yaml
+	sed -i "s/%APP_VERSION%/$(IMAGE_TAG)/g" $(CHART_DIR)/Chart.yaml
+	@echo "Preparing docs"
+	docker run --user $(shell id -u):$(shell id -g) -v $(shell pwd)/$(CHART_DIR):/helm-docs jnorwood/helm-docs:v1.11.0 -s file
+	@echo "Charts available at: $(CHART_DIR)"
+
+install: build
+	$(HELM) install gadget $(CHART_DIR) --namespace gadget --create-namespace
```

Comment:
Maybe it would make sense to use `helm upgrade --install` + `kubectl apply` for CRDs (as `helm upgrade` does not update CRDs) to be able to iterate on chart when developing it? `install` is not idempotent.